### DISCUSSION TITLE
Paperless-ngx is not using the configured PostgreSQL

### DISCUSSION THREAD

**Discussion by giacomo892** [tteck/Proxmox#1716] [General]
The paperless-ngx install script sets up and configure paperless to use a PostgreSQL database.

Looks like it is completely ignored by the application which somehow still defaults to an sqlite database located in /data/db.sqlite3.

Do you have an idea why is that?

**Comment by tteck** [comment]
🤷🏻‍♂️  I'll need to investigate further.

**Comment by giacomo892** [comment]
@tteck I've further investigated, the only needed modification during the installation is to uncomment the PAPERLESS_DBHOST=localhost line in the paperless.conf file, then postgre is used as db backend as per documentation:

```
PAPERLESS_DBHOST=<hostname>
By default, sqlite is used as the database backend. This can be changed here.

Set PAPERLESS_DBHOST and another database will be used instead of sqlite.
```
Thanks for what you are doing for the community.
